README.md

```diff
@@ -3,26 +3,26 @@
 The OpenJS Foundation hosts regular summits for active contributors to the foundation projects, this repository is being used to organize them.
 
 ## Upcoming Events
-- [June 22, 25 & 26, 2020 - ~Austin, Texas, USA~ Online](https://github.com/openjs-foundation/summit/issues/236) - [Register Here](https://www.cvent.com/events/openjs-collaborator-summit-2020/registration-71e4281e54dd46a0a0e2da15013e52db.aspx?_ga=2.164120568.674445174.1588801738-1126120395.1571424999&fqp=true) or [Register for OpenJS World](https://events.linuxfoundation.org/openjs-world/)
+- [June 9, 10, 2022 - Austin, Texas, USA](https://github.com/openjs-foundation/summit/issues/310)
 
-__Note:__ The Collab Summit is now an online-only event, per the current [Novel Coronavirus Update](https://events.linuxfoundation.org/openjs-world/attend/novel-coronavirus-update/).
-
-To better adjust for a worldwide, online event we have added an additional day of programming. 
+## Registration
+- You can sign-up for Collab Summit when [registering for OpenJS World](https://events.linuxfoundation.org/openjs-world/)
+- If you are not attending OpenJS World 2022, and just want to register for Collab Summit, please reach out to [EMAIL].
 
 ## Planning Meetings
 
 Every 2 weeks on Tuesday - see [issues](https://github.com/openjs-foundation/summit/issues) for meeting notices and agendas
 
-6:00 – 7:00pm UTC 
+15:00 – 16:00 UTC
 
 Join via Zoom: https://zoom.us/j/742726725 - Everyone is welcome!
 
 ## Cost
-Free!
 
-Although the summits may be coordinated with other events/conferences, you do not need to pay/attend those events to participate in a summit. We also have travel funds for individual members of the Node.js foundation (e.g. Node.js core collaborators) to cover their travel expenses. See [the documentation on travel funds](https://github.com/openjs-foundation/cross-project-council/tree/HEAD/proposals/approved/TRAVEL_FUND#openjs-travel-fund) for details.
+Registration for summit can be done when you [register for OpenJS World](https://events.linuxfoundation.org/openjs-world/), without any additional cost. If you are not attending OpenJS World and just want to attend Collab Summit then there is fee of 15$ USD. We also have travel funds for the participants of OpenJS projects to cover their travel expenses. See [the documentation on travel funds](https://github.com/openjs-foundation/cross-project-council/blob/main/project-resources/MEMBER_TRAVEL_FUND.md) for details.
```

Comment:
Did we discuss this fee? I only ask because it may not be easy to collect such a fee. I just want to be sure.

Reply:
Good point. I have asked in that channel. Will wait for the confirmation.